.test/test.sh

```diff
@@ -12,5 +12,27 @@ set -- docker:cli docker:dind docker:windowsservercore notary # a little bit of
 # (see "library/" and ".external-pins/" for where these come from / are hard-coded for consistent testing purposes)
 # NOTE: we are explicitly *not* pinning "golang:1.19-alpine3.16" so that this also tests unpinned parent behavior (that image is deprecated so should stay unchanging)
 
+time bashbrew fetch "$@"
+
 time "$dir/../sources.sh" "$@" > "$dir/sources.json"
+
 time "$dir/../builds.sh" --cache "$dir/cache-builds.json" "$dir/sources.json" > "$dir/builds.json"
+
+# generate an "example commands" file so that changes to generated commands are easier to review
+jq -r -L "$dir/.." '
+	include "meta";
+	[
+		first(.[] | select(normalized_builder == "buildkit")),
+		first(.[] | select(normalized_builder == "classic")),
+		first(.[] | select(normalized_builder == "oci-import")),
```

Comment:
It seems like this would select three builds, but there are only two sets of commands generated. Is this because there are no oci-import examples yet?

Reply:
Yes, that's correct -- I had originally left a `# TODO oci-import` here, but realized I could just add the conditional and it would match as soon as there was an example in the test data (which I plan to add when I have an implementation in `meta.jq`).

Reply:
(In other words, this is currently a very fancy and computational way to spell `empty` :joy:)